src/Tasks/Microsoft.NET.Build.Tasks/FrameworkPackages/FrameworkPackages.netcoreapp2.1.cs

```diff
@@ -17,23 +17,23 @@ internal static class NETCoreApp21
             { "Microsoft.NETCore.App", "2.1.0" },
             { "Microsoft.VisualBasic", "10.3.0" },
             { "Microsoft.Win32.Registry", "4.5.0" },
-            { "System.Buffers", "4.5.0" },
+            { "System.Buffers", "4.6.1" },
             { "System.Collections.Immutable", "1.5.0" },
             { "System.ComponentModel.Annotations", "4.5.0" },
             { "System.Diagnostics.DiagnosticSource", "4.5.0" },
             { "System.IO.FileSystem.AccessControl", "4.5.0" },
             { "System.IO.Pipes.AccessControl", "4.5.0" },
-            { "System.Memory", "4.5.5" },
-            { "System.Numerics.Vectors", "4.5.0" },
+            { "System.Memory", "4.6.3" },
+            { "System.Numerics.Vectors", "4.6.1" },
             { "System.Reflection.DispatchProxy", "4.5.0" },
             { "System.Reflection.Metadata", "1.6.0" },
             { "System.Security.AccessControl", "4.5.0" },
             { "System.Security.Cryptography.Cng", "4.5.2" },
             { "System.Security.Cryptography.OpenSsl", "4.5.0" },
             { "System.Security.Principal.Windows", "4.5.0" },
             { "System.Threading.Tasks.Dataflow", "4.9.0" },
-            { "System.Threading.Tasks.Extensions", "4.5.4" },
-            { "System.ValueTuple", "4.5.0" },
+            { "System.Threading.Tasks.Extensions", "4.6.3" },
```

Comment:
We cannot change https://github.com/dotnet/sdk/blob/0e185be17c129e3b3b47fa9688a7a5dd882fca5d/src/Tasks/Microsoft.NET.Build.Tasks/FrameworkPackages/FrameworkPackages.netcoreapp2.0.cs#L207 because `netcoreapp2.0` only included assembly `4.1.1.0` while the package has `4.2.1.0`